Issue #203 Add hardware section - added link &tc (#218)
wpilibsuite/frc-docs: source/docs/hardware/getting-started/wiring-pneumatics.rst

Wiring Pneumatics
=================

> .. hint:: For pneumatics safety & mechanical requirements, consult this year's Robot Construction rules (Section 10 in 2019). For mechanical design guidelines, the FIRST Pneumatics Manual is located `here (last updated 2017)<https://firstfrc.blob.core.windows.net/frc2017/pneumatics-manual.pdf>`__

Reviewer: ```suggestion
.. hint:: For pneumatics safety & mechanical requirements, consult this year's Robot Construction rules (Section 10 in 2019). For mechanical design guidelines, the FIRST Pneumatics Manual is located `here (last updated 2017) <https://firstfrc.blob.core.windows.net/frc2017/pneumatics-manual.pdf>`__
```

Wiring Overview
---------------

A single PCM will support most pneumatics applications, providing an output for the compressor, input for the pressure switch, and outputs for up to 8 solenoid channels (12V or 24V selectable). The module is connected to the roboRIO over the CAN bus and powered via 12V from the PDP.

For complicated robot designs requiring more channels or multiple solenoid voltages, additional PCMs can be added to the control system.

PCM Power and Control Wiring
----------------------------

> The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM see Wiring the 2015 FRC Control System. 

Reviewer: ```suggestion
The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM see Wiring the 2015 FRC Control System.
```

> The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM see Wiring the 2015 FRC Control System.

Reviewer 2: Needs a link to the wiring document here.

Author: Added based on current location of Wiring the Control System document, hopefully it will stay intact?

> The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM, see `How To Wire A Robot <https://docs.wpilib.org/en/latest/docs/hardware/getting-started/how-to-wire-a-robot.html>`__

Reviewer: ```suggestion
The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM, see :doc:`How To Wire A Robot <how-to-wire-a-robot>`
```

> The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM, see `How To Wire A Robot <https://docs.wpilib.org/en/latest/docs/hardware/getting-started/how-to-wire-a-robot.html>`__ 

Reviewer: ```suggestion
The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM, see :ref:`How To Wire A Robot <docs/hardware/getting-started/how-to-wire-a-robot:Pneumatics Control Module Power (Optional)>`
```

> The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM, see `How To Wire A Robot <https://docs.wpilib.org/en/latest/docs/hardware/getting-started/how-to-wire-a-robot.html>`__ 

Reviewer: ```suggestion
The first PCM on your robot can be wired from the PDP VRM/PCM connectors on the end of the PDP. The PCM is connected to the roboRIO via CAN and can be placed anywhere in the middle of the CAN chain (or on the end with a custom terminator). For more details on wiring a single PCM, see `How To Wire A Robot <https://docs.wpilib.org/en/latest/docs/hardware/getting-started/how-to-wire-a-robot.html>`__
```
